industrial_moveit_benchmarking/launch/stomp_valgrind.launch

```diff
@@ -3,6 +3,7 @@
   <arg unless="$(arg profile)" name="launch_prefix" value="" />
   <arg     if="$(arg profile)" name="launch_prefix" value="valgrind --tool=callgrind" />
 
-  <rosparam command="load" file="$(find stomp_test_kr210_moveit_config)/config/stomp_config.yaml" />
-  <node name="stomp_benchmarking_node" pkg="industrial_moveit_benchmarking" type="stomp_benchmarking_node" launch-prefix="$(arg launch_prefix)"/>
+  <rosparam command="load" file="$(find industrial_moveit_benchmarking)/resources/config/stomp_config.yaml" />
```

Comment:
Same question as for `cartesian_valgrind.launch`.

Reply:
I ran into the same issue where the node wasn't loading the parameters from the private namespace.